src/chart/options/channels.cpp

```diff
@@ -3,14 +3,15 @@
 using namespace Vizzu;
 using namespace Vizzu::Gen;
 
-Channels::Channels()
-{
-	for (auto type = 0U; type < std::size(channels); type++)
-		channels[static_cast<ChannelId>(type)] =
-		    Channel::makeChannel(ChannelId(type));
-
-	reset();
-}
+Channels::Channels() :
```

Comment:
What is the benefit of this pattern over a simple loop?

Reply:
Because this is an array initialization where the elements are not default constructible from now.